appengine-flexible-archetype/src/test/resources/projects/basic/archetype.properties

```diff
@@ -3,6 +3,4 @@ package=it.pkg
 version=0.1-SNAPSHOT
 groupId=archetype.it
```

Comment:
Update to?
`package` to com.example.flexible_archetype
`groupId` to com.example.flexible_archetype